script/utils/start-backing-services

```diff
@@ -1,40 +1,30 @@
 #!/bin/sh
 
-echo "==> Starting the backing services in Docker..."
-
 for arg in "$@"
 do
   if [[ $arg == "--mock-api" ]]; then
     MOCK_API=true
+    DOCKER_PARAM="--scale=hmpps-accredited-programmes-api=0 --scale=hmpps-accredited-programmes-db=0"
   fi
 done
 
-if [ "$MOCK_API" = true ]; then
-  docker compose up --scale=hmpps-accredited-programmes-api=0 --scale=postgresql=0 -d
+echo "==> Starting the backing services in Docker..."
+docker compose up $DOCKER_PARAM -d
 
-  echo "==> Stubbing API endpoints..."
+npx ts-node --transpile-only ./wiremock/scripts/resetStubs.ts > /dev/null
```

Comment:
Might be worth adding an echo to say `Resetting API stubs...` above this. In case anything fails, it'll help us pinpoint whether it's a Docker or Wiremock issue